package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zbayapp/identity",
-  "version": "1.0.11",
+  "version": "1.0.12",
```

Comment:
You're introducing required parametr, so this is a breaking change. 
Probably this should update major, so that we don't screw up a project that consumes this

Reply:
sure